src/transformers/models/maest/modeling_maest.py

```diff
@@ -0,0 +1,42 @@
+# coding=utf-8
```

Comment:
As mentioned by @NielsRogge - we can skip adding a modelling file since it's one-to-one the same as AST, and promote the `AutoModel` API

Reply:
Addressed in ea1c43e